sycl/source/detail/device_info.hpp

```diff
@@ -838,8 +838,8 @@ struct get_device_info_impl<
       };
     else if ((architecture::intel_gpu_pvc == DeviceArch) ||
              (architecture::intel_gpu_bmg_g21 == DeviceArch) ||
-             (architecture::intel_gpu_lnl_m == DeviceArch))
-      return {
+             (architecture::intel_gpu_lnl_m == DeviceArch)) {
```

Comment:
I don't see functional change to device_info.hpp. Original code was using the same style in each if/else branch.
With the change now one branch is using different style. I don't see why this change is needed. Please, revert it.